INSTALL.md

```diff
@@ -99,13 +114,42 @@ The initialization command will prompt for the following information.
    prompts for its password.  This information will be needed to later connect
    to Accumulo.
 
-Multiple Nodes
---------------
+## Run Accumulo
+
+There are two primary methods for running Accumulo:
+
+1. Run individual Accumulo services using `accumulo-service`. Useful if you are
+   using a cluster management tool (i.e Ansible, Salt, etc) to start Accumulo.
```

Comment:
Depends on the init system. Most of what these scripts do are handled within modern init systems. These might be useful for SysVinit, but certainly not for systemd. I'm not sure about upstart, and others. In systemd, I've found it better to call "accumulo tserver" directly, and it handles tracking the service. So, it's best to use the simplest, least side-effect-y scripts available.